Source/Engine/Level/Actors/AnimatedModel.h

```diff
@@ -9,6 +9,14 @@
 #include "Engine/Renderer/DrawCall.h"
 #include "Engine/Core/Delegate.h"
 
+API_STRUCT() struct ModelBoneNode
```

Comment:
I would name it `NodeTransformation`  and move it inside `AnimatedModel` as nested struct. Then add simple docs comments.